src/terrasoApi/user/userFragments.ts

```diff
@@ -15,7 +15,7 @@
  * along with this program. If not, see https://www.gnu.org/licenses/.
  */
 
-export const userFields = `
+export const userFields = /* GraphQL */ `
```

Comment:
Question: What is the `/* GraphQL */` doing?

Reply:
it marks the code for inclusion in the graphql validation & type generation, i will explain more in another comment